### PR TITLE
fix: broken link

### LINKED ISSUE #5378: Broken link for C++ cheat sheet : https://www.hoomanb.com/cs/quickref/CppQuickRef.pdf
The C++ cheat sheet PDF link is broken.

https://www.hoomanb.com/cs/quickref/CppQuickRef.pdf

### PR DESCRIPTION
**Pull request by rkapoor10**
Fixes #5378
## What does this PR do?
 Adds info | Improve repo

### CODE REVIEW

**Comment by eshellman** [comment]
unauthorized pdfs hosted on personal homepages are likely to go bad, just as the previous one went bad. I'd just remove the link.

**Review by davorpa — CHANGES_REQUESTED** [review]
Change from HTTPS to HTTP

Resolves EbookFoundation/free-programming-books#5378

File: more/free-programming-cheatsheets.md [hunk 1]
```diff
@@ -40,7 +40,7 @@
 
 <h3 id="cpp">C++</h3>
 
-* [C++ Quick Reference](https://www.hoomanb.com/cs/quickref/CppQuickRef.pdf) (PDF)
+* [C++ Quick Reference](https://www.cs.ccu.edu.tw/~damon/oop/,c++refcard.pdf) (PDF)
```
Comment (davorpa) [inline more/free-programming-cheatsheets.md:43]: - Change initial url from `https` to `http`
- Give credits to authors

```suggestion
* [C++ Quick Reference](http://www.hoomanb.com/cs/quickref/CppQuickRef.pdf) - Hooman Baradaran (PDF)
```

**Review by davorpa — CHANGES_REQUESTED** [review]

**Comment by eshellman** [comment]
please resolve conflicts.

**Pull request merged**